src/main/java/TechJobs.java

```diff
@@ -120,6 +117,17 @@ private static String getUserSelection(String menuHeader, HashMap<String, String
     // Print a list of jobs
     private static void printJobs(ArrayList<HashMap<String, String>> someJobs) {
 
-        System.out.println("printJobs is not implemented yet");
+        for (HashMap<String, String> job : someJobs) {
+            System.out.println("\n*****");
+            for (Map.Entry<String, String> columnKey : job.entrySet()) {
+                System.out.println(columnKey.getKey() + ": " + columnKey.getValue());
+            }
+            System.out.println("*****");
+        }
+
+        if (someJobs.size() == 0) {
+            System.out.println("No Results\n");
+        }
```

Comment:
```
        if (someJobs.size() == 0) {
            System.out.println("No Results\n");
            return;
        }
        
        for (HashMap<String, String> job : someJobs) {
            System.out.println("\n*****");
            for (Map.Entry<String, String> columnKey : job.entrySet()) {
                System.out.println(columnKey.getKey() + ": " + columnKey.getValue());
            }
            System.out.println("*****");
        }
```

Great work with printing the jobs in Assignment 1! The only thing in TechJobs.java I would recommend is moving you if statement above your for loop and adding a return at the end of the if block. You could also use an if-else statement in this case. It doesn't really make any difference in output or performance (I believe), but perhaps if a for loop had more logic, it may probably be better to avoid hitting those lines of code if it's empty.